### ISSUE TITLE
Adding foe as a unit to astropy.units

### ISSUE THREAD

**Issue by johntkappel** [astropy/astropy#13459]
### Description

The foe is an uncommon unit of energy used in astrophysics, which is equivalent to 10**51 ergs (**F**ifty-**O**ne **E**rgs - FOE). It is equivalent to the energy released by a supernova, and so is occasionally used. It is also referred to as a Bethe, though this is less common. 

Example of implementation would be:
`
def_unit(['Foe', 'foe','Bethe','bethe'], cgs.erg ** 51, namespace=_ns, prefixes=True)`

### Additional context

Either units.cgs or units.misc would be the appropriate place to implement this feature.

The term was originally proposed by Hans Bethe. 
Examples of using foe and bethe as units: 
https://permalink.lanl.gov/object/tr?what=info:lanl-repo/lareport/LA-UR-97-2534-14
https://www.sciencedirect.com/science/article/pii/0375947496002345
https://iopscience.iop.org/article/10.3847/1538-4357/abf82e/meta

**Comment by mhvk** [comment]
Yes, I think that would be fine to add. Probably best in `astrophys,py`. PR welcome, but note that the definition is not quite right: should be `1e51 * erg`. Also, at least for `foe`, there should be no prefixes; I think this would be best for `Bethe` too. And I would not include "Foe" with a capital - I've only seen it in all lower case.

**Comment by JostMigenda** [comment]
I stumbled across this issue because having foe in `astropy.units` would be useful in our supernova code [snewpy](https://github.com/SNEWS2/snewpy). I’ll try to submit a PR later today or tomorrow.
